### ISSUE TITLE
Union types?

### ISSUE THREAD

**Issue by pdesaulniers** [hishamhm/tl#40]
One of the overloads of love.graphics.print expects a table that looks like this:
```lua
love.graphics.print({{1, 1, 1, 1}, "Hello", {1, 0, 0, 1}, " World"})
```

I tried to declare this function, but I get a syntax error:
```lua
global love_graphics = record
	print: function(coloredtext: {{number} or string})
end

global love = record
	graphics: love_graphics
end
```
Should this syntax for union types work?

**Comment by hishamhm** [comment]
Union types are not supported at this point. Unfortunately, you'll have to declare this as `{any}` for now.

They might be coming in the future, so let's leave this issue open!

**Comment by hishamhm** [comment]
@pdesaulniers  Please take a look at #63  !

**Comment by hishamhm** [comment]
Merged #63!
